### PR TITLE
Reservations report

### PR DESCRIPTION
**Pull request by eliasjuanpablo**
Three reports were added:

- Reservations by event: top 10 events ordered by reservation count.
- Reservations by month: each month with their reservation count.
- Attendees languages: each language with the their reservation count.

Notes: 
- Some noisy changes regarding doble quotes were added :eyes:
- Some choice fields had to be refactored to make it easier to show readable values on reports.

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -419,3 +502,46 @@ def approve(self, request, pk):
...
+                .annotate(language=F("user__language"))
+                .annotate(count=Count("id"))
+                .values("language", "count")
+                .filter(event_occurrence__event__in=user_events)
+            ),
+        }
+
+        # Awful hack to return months with count = 0
+        # To achieve this with SQL, a Month table is needed :(
+        zero_count_months = [{"month": i, "count": 0} for i in range(1, 13)]
```
Comment (juampivallejo) [inline api/views.py]: Doesn't  chart js solve this for us?. It's ok though

File: api/models.py [hunk 2]
```diff
@@ -109,7 +109,7 @@ class Event(models.Model):
     name = models.CharField(max_length=80)
     description = models.TextField()
     cancelled = models.DateTimeField(null=True, blank=True)
-    winery = models.ForeignKey(Winery, on_delete=models.PROTECT)
+    winery = models.ForeignKey(Winery, on_delete=models.PROTECT, related_name='events')
```
Comment (juampivallejo) [inline api/models.py]: nit: this is not really needed, we can use event_set (just learned it.)

File: api/views.py (hunk 1 above)
Comment (eliasjuanpablo) [reply api/views.py]: Sadly, it doesn't. :(

File: api/models.py (hunk 2 above)
Comment (eliasjuanpablo) [reply api/models.py]: Interesting. I think I like this approach because it's more explicit, but great catch!

**Pull request merged**